graphql.schema.json

```diff
@@ -0,0 +1,2082 @@
+{
```

Comment:
Do we want to check this schema in? From the PR description it sounded like we would want to point `SCHEMA_PATH` either to schema file generated in the warehouse repo, or (later, maybe?) to a URL.

Reply:
Oh okay, gotcha. I think we can remove (or comment out) the `./graphql.schema.json` step in `codegen.yml` if we don't have a use for it yet

Reply:
@gigxz Agreed